Main/ConsistencyCheck.h

```diff
@@ -31,6 +31,7 @@ class NameToCheck {
 					case item_new_name:	return fFileListItem->fNewName;
 					case item_disk_name: return fName;
 				}
+				return NULL;
```

Comment:
you can replace "return NULL;" with "abort();" 

it should fix the warning because abort() never returns